### ISSUE TITLE
ngcc parallel issue

### ISSUE THREAD

**Issue by farfromrefug** [lerna/lerna#3028]
I am facing an issue with `lerna run` and `ng-packagr`. It seems like multiple processes are run in the same time.
ngcc reports and error about multiple processes run at the same time.

Though if i run the npm command separately in each package it works.
While the `lerna run` command is running i do see 9 ngcc processes running. I see only one while running on a single package

## Expected Behavior
`lerna run build.angular` should succeed

## Current Behavior
```
npm run build.angular

> @nativescript-community/ui-material-components@1.0.2 build.angular
> lerna run build.angular

lerna notice cli v4.0.0
lerna info Executing command in 14 packages: "npm run build.angular"
lerna ERR! npm run build.angular exited 1 in '@nativescript-community/ui-material-cardview'
lerna ERR! npm run build.angular stdout:

> @nativescript-community/ui-material-cardview@6.2.3 build.angular
> ../../node_modules/.bin/ng-packagr -p ../../src/cardview/angular/package.json -c ../../src/cardview/angular/tsconfig.json

Building Angular Package

------------------------------------------------------------------------------
Building entry point '@nativescript-community/ui-material-cardview-angular'
------------------------------------------------------------------------------
lerna ERR! npm run build.angular stderr:
- Compiling with Angular sources in Ivy full compilation mode.
Another process, with id 84977, is currently running ngcc.
Waiting up to 250s for it to finish.
(If you are sure no ngcc process is running then you should delete the lock-file at /Volumes/dev/nativescript/nativescript-material-components/node_modules/.pnpm/@angular+compiler-cli@13.0.1_ffb0fe389b9d01a46dfa28711196672c/node_modules/@angular/compiler-cli/bundles/ngcc/__ngcc_lock_file__.)
✖ Compiling with Angular sources in Ivy full compilation mode.
ngcc is already running at process with id 84979.
If you are running multiple builds in parallel then you might try pre-processing your node_modules via the command line ngcc tool before starting the builds.
(If you are sure no ngcc process is running then you should delete the lock-file at /Volumes/dev/nativescript/nativescript-material-components/node_modules/.pnpm/@angular+compiler-cli@13.0.1_ffb0fe389b9d01a46dfa28711196672c/node_modules/@angular/compiler-cli/bundles/ngcc/__ngcc_lock_file__.)
lerna ERR! npm run build.angular exited 1 in '@nativescript-community/ui-material-cardview'
lerna WARN complete Waiting for 6 child processes to exit. CTRL-C to exit immediately.
```

## Steps to Reproduce (for bugs)
clone `https://github.com/nativescript-community/ui-material-components`
run `npm i`
run `npm run build.angular`

<details><summary>lerna.json</summary><p>

```json
{
    "packages": [
        "packages/*"
    ],
    "npmClient": "npm",
    "command": {
        "publish": {
            "conventionalCommits": true,
            "ignoreChanges": [
                "ignored-file",
                "*.md"
            ]
        },
        "bootstrap": {
            "npmClientArgs": [
                "--no-package-lock"
            ]
        }
    },
    "version": "6.2.3"
}

```
</p></details>

## Context
build mono repo packages 

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 4.0.0 |
| `npm --version`  | 8.1.3 |
| `node --version` | v17.0.1 |

| OS | Version |
| --- | --- |
| macOs | 11.6 |

**Comment by tujlaky** [comment]
This fixed my issue:
https://github.com/angular/angular/issues/38875#issuecomment-693634786

**Comment by JamesHenry** [comment]
Hi Folks 👋 

You may or may not know that lerna is now under the stewardship of Nrwl (announcement here https://github.com/lerna/lerna/issues/3121), a company with a long history of not just producing valuable open-source software (OSS), but also backing others (at the time of writing, Nrwl has donated over $50,000 to OSS it _hasn't_ created, see https://opencollective.com/nx for full details).

Quite simply, Nrwl ❤️  OSS, and is committed to making lerna the best it can be. We use it ourselves.

We hope you will continue to be a part of this community as we look to take things forward from here!

Please see https://github.com/lerna/lerna/discussions/3140 for more details on our plans for 2022.

In the case of this specific issue, it looks like it was down to a limitation in ngcc which has since been patched.

If you run into any issues on the latest version of lerna, please feel free to open a new issue and follow the instructions:
https://github.com/lerna/lerna/issues/new/choose

Many thanks 🙏
